openapi/zoho.crm.rest/Ballerina.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Sales & CRM/Customer Relationship Management", "Cost/Paid"]
+org = "ballerinax"
+name = "zoho.crm.rest"
+icon = "icon.png"
+distribution = "2201.0.3"
+repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
+version = "1.0.1"
```

Comment:
Shall we go with 1.0.0. Rebuild after doing the change.

Reply:
Fixed in https://github.com/ballerina-platform/openapi-connectors/commit/ff0eb5a6dd70aefdc38bb071a3152c9930f30b9f